qoqo/src/operations/analog_operations.rs

```diff
@@ -21,8 +21,6 @@ use roqoqo::operations::*;
 use struqture::spins::SpinHamiltonian;
 use struqture_py::spins::SpinHamiltonianSystemWrapper;
 
-#[cfg(feature = "json_schema")]
-use roqoqo::ROQOQO_VERSION;
```

Comment:
Are we sure this is not needed even with the json_schema feature?

Reply:
`rustc` flags it as unused import, testing the feature shows no problems